idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -1108,8 +1110,8 @@ def fetch_references():
         lineage_paths[0] = fetch_reference(LINEAGE_SHELF)
         accession2taxid_paths[0] = fetch_reference(ACCESSION2TAXID)
 
-    reference_fetcher_thread = threading.Thread(target=fetch_references)
-    reference_fetcher_thread.start()
+    t = threading.Thread(target=fetch_references)
```

Comment:
What's the rationale for removing the descriptive name `reference_fetcher_thread`? :)